notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>Apply these changes to the "Deploy an XRAI model" cell, as well.</p><p> </p><p>Change first paragraph, second sentence: "There are two <code>gcloud</code> arguments for explainability:"</p><p> </p><p>Check for correct spelling of "explainability" throughout the file. The misspelling here was "explanability"</p><p> </p><p><code>--explanation-method</code>&nbsp;: Specifies which explainability method to use for explanations -- in this case, integrated gradients.</p><p> </p><p>Also Ctrl-F for "approxiamation" / "approxiamate" and change it to "approximation" / "approximate"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>